demisto_sdk/commands/format/format_module.py

```diff
@@ -47,11 +49,11 @@ def format_manager(input: str = None, output: str = None, from_version: str = No
         int 0 in case of success 1 otherwise
     """
     if input:
-        files = get_files_in_dir(input, ['json', 'yml'])
+        files = get_files_in_dir(input, ['json', 'yml', 'py'])
     else:
         files = [file['name'] for file in
                  get_changed_files(filter_results=lambda _file: not _file.pop('status') == 'D')]
-    if output and not output.endswith(('.yml', 'json')):
+    if output and not output.endswith(('.yml', 'json', 'py')):
```

Comment:
why `yml` has a dot and the other two don't?

Reply:
no reason,
changed.